.github/workflows/coccicheck.yaml

```
@@ -1,7 +1,10 @@
 name: BPF checks
 
 on:
-  - pull_request
+  pull_request: []
```

```suggestion
  pull_request: {}
```
The "fluid schema" can be a little misleading...  in this case it's object, not an array.

The logic is that there is shortcut syntax, like this:
```YAML
on: [ key1, key2 ]
```
And long syntax:
```YAML
on:
 key1: {},
 key2:
   branches: [ master ]
```

---

Hm... looks like `pull_request: []`  actually passed, perhaps the parser is just very lazy! Do you mind changing anyway?
The parser probably looks for keys, and since `pull_request` maps to nothing, it ignores the type of its value.

---

Makes sense. I've fixed it.